app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="animal age is 0"
+        ),
+        pytest.param(
+            10, 10, [0, 0],
+            id="animal age less than 15"
+        ),
+        pytest.param(
+            16, 19, [1, 1],
+            id="animal age is more 15 and less 23"
+        ),
+        pytest.param(
+            23.9, 23.1, [1, 1],
+            id="animal age with remainder"
+        ),
```

Comment:
you need to test 14, 15, 23, 24 years for both cats and dogs, because these values are edge cases